boxoffice/views/admin.py

```diff
@@ -24,11 +25,26 @@ def index():
     return dict(user=g.user)
 
 
+def jsonify_ic(ic):
```

Comment:
Since this returns a dictionary and not JSON, the name should be `as_dict`. Also, this should be a method on the ItemCollection, not an independent function.

Reply:
Actually, you don't even need this. RoleMixin's `access_for` should construct the dictionary for you. @shreyas-satish can you help with this?

Reply:
```
class ItemCollection(BaseScopedNameMixin, db.Model):
    # ... excluded table details
    __roles__ = {
        'ic_owner': {
            'write': {},
            'read': {'id', 'name', 'title'}
        }
    }

    def roles_for(self, user=None, token=None):
        if not user and not token:
            return set()
        roles = super(ItemCollection, self).roles_for(user, token)
        if user or token:
            roles.add('user')
        if self.organization.userid in user.organizations_owned_ids():
            roles.add('ic_owner')
        return roles

Example usage:
=> ic = ItemCollection.query.first()
=> dict(ic.access_for(user=g.user))
```

Reply:
Fixed